sortedcontainers/sortedset.pyi

```diff
@@ -0,0 +1,81 @@
+import typing
+from typing import (
+    Any,
+    AbstractSet,
+    Callable,
+    Generic,
+    Hashable,
+    Iterable,
+    Iterator,
+    List,
+    MutableSet,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
+    Set,
+    TypeVar,
+    Union,
+)
+
+# --- Global
+
+_T = TypeVar("_T", bound=Hashable)
+_S = TypeVar("_S", bound=Hashable)
+_SS = TypeVar("_SS", bound="SortedSet")
```

Comment:
`SortedSet` doesn't need quotes around it. In `pyi` files, forward references work without quotes.